package.json

```diff
@@ -6,8 +6,8 @@
   },
   "license": "BSD-3-Clause",
   "engines": {
-    "node": ">=14 <15",
-    "npm": ">=6.11.0 <7"
+    "node": ">=16 <17",
```

Comment:
Is it fine to suddenly update this?

Reply:
TBH I don't know on how many environments we follow that constrain.
In our Github Actions more than half of them uses Node@16 already (I tried to fix that in different pull request, that wasn't merged yet), I'm going to guess, that most of engineers uses Node@16 or greater to build this project. The only place that we hanve Node@14 was Vercel itself, but not anymore (as @timuric upgraded that before I created this pull request).